motion/.npmignore

```diff
@@ -0,0 +1,7 @@
+android/.gradle
+ios/Pods
+ios/Podfile.lock
```

Comment:
motion is web only, so doesn't have any of this

Reply:
I considered this, but I'm fine specializing it